### PR TITLE
Use Queues maxsize to limit execution.

### PR DESCRIPTION
**Pull request by La0**
No more `is_full`, nor `nb_messages`

### CODE REVIEW

File: src/pulselistener/pulselistener/lib/bus.py [hunk 1]
```diff
@@ -12,30 +12,22 @@ class MessageBus(object):
...
         Create a new queue on the message bus
         * asyncio by default
         * multiprocessing when mp=True
+        By default, there are no size limit enforced (maxsize=-1)
         '''
         assert name not in self.queues, 'Queue {} already setup'.format(name)
+        assert isinstance(maxsize, int)
         if mp:
-            self.queues[name] = multiprocessing.Queue()
+            self.queues[name] = multiprocessing.Queue(maxsize=maxsize)
```
Comment (marco-c) [inline src/pulselistener/pulselistener/lib/bus.py]: In the multiprocessing docs, it's not specified what happens when maxsize < 0, but I guess it's the same behavior as asyncio.Queue. Maybe you can confirm with a simple script to put a lot of things into a multiprocessing.Queue and see that it never blocks.

Comment (La0) [reply src/pulselistener/pulselistener/lib/bus.py]: It says [here](https://docs.python.org/3.5/library/multiprocessing.html#multiprocessing.Queue):
> Queue implements all the methods of queue.Queue except for task_done() and join().

So i took that as the same behaviour was implemented.

I'll add the test though, to be sure

Comment (marco-c) [reply src/pulselistener/pulselistener/lib/bus.py]: I meant just testing it locally on your machine to confirm, but we don't need to add a test here

Comment (La0) [reply src/pulselistener/pulselistener/lib/bus.py]: Oups.

Comment (marco-c) [reply src/pulselistener/pulselistener/lib/bus.py]: Well, it won't hurt ;)

**Pull request merged**